controller/health.go

```diff
@@ -38,7 +42,10 @@ func setApplicationHealth(resources []managedResource, statuses []appv1.Resource
 			}
 			healthStatus, err = health.GetResourceHealth(res.Live, healthOverrides)
 			if err != nil && savedErr == nil {
-				savedErr = err
+				errCount++
+				savedErr = fmt.Errorf("failed to get resource health for %q with name %q in namespace %q: %w", res.Live.GetKind(), res.Live.GetName(), res.Live.GetNamespace(), err)
+				// also log so we don't lose the message
+				log.WithField("application", app.QualifiedName()).Error(savedErr)
```

Comment:
if `savedErr` is logged outside of this function, we would be duplicating the log entries. Can you please double check if it would be ok to log at this level?

Reply:
This message currently appears in an info-level log in `normalizeApplication` which summarizes the changes being made to the application spec (in this case, adding an error condition).

But in that case, we're only logging the last error encountered. I think logging the resource-specific error as a warning would be much more helpful for debugging. 

I think the light duplication is acceptable, because the existing logging only accidentally includes this information.

Reply:
ok. but this is not at warning level. I agree with you I think we should change it to warning.

Reply:
oh ha. thanks!